sdk/python/feast/repo_config.py

```diff
@@ -37,6 +37,26 @@ class DatastoreOnlineStoreConfig(FeastBaseModel):
 OnlineStoreConfig = Union[DatastoreOnlineStoreConfig, SqliteOnlineStoreConfig]
 
 
+class FileOfflineStoreConfig(FeastBaseModel):
+    """ Offline store config for local (file-based) store """
+
+    type: Literal["file"] = "file"
+    """ Offline store type selector"""
+
+
+class BigqueryOfflineStoreConfig(FeastBaseModel):
```

Comment:
`BigqueryOfflineStoreConfig` -> `BigQueryOfflineStoreConfig` (inconsistent with how we have been typing it)